docker/quickstart/docker-compose-without-neo4j-m1.quickstart.yml

```diff
@@ -16,33 +16,23 @@ services:
     - KAFKA_HEAP_OPTS=-Xms256m -Xmx256m
     - KAFKA_CONFLUENT_SUPPORT_METRICS_ENABLE=false
     hostname: broker
-    image: confluentinc/cp-kafka:7.2.2
+    image: kymeric/cp-kafka:latest
```

Comment:
This is not correct, this repo is like 3 years old at this point and the confluentinc/cp-kafka has support for arm now.